src/Tests/Microsoft.NET.TestFramework/ProjectConstruction/TestProject.cs

```diff
@@ -426,35 +432,5 @@ private bool ReferenceAssembliesAreInstalled(string targetFrameworkVersion)
             var requestedReferenceAssembliesPath = Path.Combine(new DirectoryInfo(net461referenceAssemblies).Parent.FullName, targetFrameworkVersion);
             return Directory.Exists(requestedReferenceAssembliesPath);
         }
-
-        public override string ToString()
```

Comment:
I assume somebody did this for debugging at some point? I think I agree it's not worth maintaining.

Reply:
That was my assumption; I looked high and low for any usages and couldn't find any.